### PR TITLE
Add tags to publish/import wkw modal

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

File: app/scripts/proactive/templates/catalog-publish-description-first.html___jb_old___ [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<div id="catalog-publish-description" data-first="true">
```
Comment (fviale) [inline app/scripts/proactive/templates/catalog-publish-description-first.html___jb_old___:1]: same comment as above

File: app/scripts/proactive/templates/catalog-publish-description.html___jb_old___ [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<div id="catalog-publish-description">
```
Comment (fviale) [inline app/scripts/proactive/templates/catalog-publish-description.html___jb_old___:1]: same comment

File: app/scripts/proactive/view/CatalogPublishView.js [hunk 3]
```diff
@@ -379,7 +389,7 @@ define(
 
             if( typeof alreadyPublishedBucketName === "undefined"){
                 var studioApp = require('StudioApp');
-                const bucketNameObject = studioApp.models.jobModel.get("Generic Info").filter(function(item){
+                const bucketNameObject = studioApp.models.jobModel.attributes["Generic Info"].filter(function(item){
```
Comment (fviale) [inline app/scripts/proactive/view/CatalogPublishView.js:392]: why did you have to change this ?

Comment (mbenguig) [reply app/scripts/proactive/view/CatalogPublishView.js:392]: Because it throwed an unknown function error. I will test again.

Comment (mbenguig) [reply app/scripts/proactive/view/CatalogPublishView.js]: Ok it works also without this update.

**Review by amouhoub — APPROVED** [review]
LGTM

**Pull request merged**